event-land/scss/_typography.scss

```diff
@@ -7,3 +9,89 @@
 body {
   font-family: $primaryFontFamily;
 }
+
+.heading-1,
+.heading-2,
+.heading-3 {
+  font-weight: $fontWeightBold;
+  text-transform: uppercase;
+  &--primary {
+    color: $tertiaryColor;
+  }
+  &--secondary {
+    color: $secondaryColor;
+  }
+}
+
+.heading-1 {
+  font-size: $fontSize7Xl;
+  @include laptop {
+    font-size: $fontSize5Xl;
+  }
+  @include mobile {
+    font-size: $fontSize4Xl;
+  }
+}
+
+.heading-2 {
+  font-size: $fontSize5Xl;
+  @include laptop {
+    font-size: $fontSize4Xl;
+  }
+  @include mobile {
+    font-size: $fontSize3Xl;
+  }
+}
+
+.heading-3 {
+  font-size: $fontSize4Xl;
+  @include laptop {
+    font-size: $fontSize3Xl;
+  }
+  @include mobile {
+    font-size: $fontSize2Xl;
+  }
+}
```

Comment:
I think it is better if you separate the responsive codes because then your code will have better readability and you will know better which part is related to which one.
And if you separate them with comments, it will be very good